packages/orbit-components/examples/BadgePrimitive/DEFAULT.json

```diff
@@ -1,6 +1,11 @@
 {
-	"imports": "import BadgePrimitive from \"@kiwicom/orbit-components/lib/BadgePrimitive\";\nimport Stack from \"@kiwicom/orbit-components/lib/Stack\";\nimport Train from \"@kiwicom/orbit-components/lib/icons/Train\";\nimport Passengers from \"@kiwicom/orbit-components/lib/icons/Passengers\";",
-	"example": "() => (\n  <Stack direction=\"column\">\n    <BadgePrimitive>Tip</BadgePrimitive>\n    <BadgePrimitive ariaLabel=\"Train\" icon={<Icons.Train />} />\n    <BadgePrimitive ariaLabel=\"4 passengers\" icon={<Icons.Passengers />}>\n      4\n    </BadgePrimitive>\n  </Stack>\n)\n",
+	"imports": "import IllustrationPrimitive from \"@kiwicom/orbit-components/lib/IllustrationPrimitive\";",
```

Comment:
This seems odd. Could you revert these changes? Otherwise this PR is ready to merge!

Reply:
The changes made to DEFAULT.json and SUFFIX.json were autogenerated  by the script `build:examples`. Do you want me to delete both files or just this one?

Reply:
Just the DEFAULT.json file. That must be the result of a problem elsewhere and shouldn't be checked in.

Reply:
Done